components/core/src/fs.rs

```diff
@@ -44,6 +44,17 @@ const SVC_PATH: &'static str = "hab/svc";
 lazy_static! {
     static ref EUID: u32 = users::get_effective_uid();
 
+    static ref MY_CACHE_PATH: PathBuf = {
```

Comment:
Is there a reason we are adding this rather than using the existing `MY_CACHE_KEY_PATH`?